core/ivy.xml

```diff
@@ -13,7 +13,7 @@
 		<dependency org="junit" name="junit" rev="4.12" conf="*"/>
         <dependency org="org.mockito" name="mockito-all" rev="1.9.5" conf="test->*"/>
 
-		<dependency org="FreePastry" name="FreePastry" rev="2.1.1"/>
+		<dependency org="FreePastry" name="FreePastry" rev="3.0.0"/>
```

Comment:
where does this revision come from? :-)

Reply:
https://github.com/rwth-acis/FreePastry/pull/1

Or do you mean why its 3.0.0 instead of 2.2.0?

Reply:
Well...kind of both I guess. A new major version would usually indicate a breaking change (which might be the case -> Release Notes).
And the V3 of FreePastry wasn't online yet, so this might have broken the build, no? I only just merged the pull request..

In any case, good work, thanks!

Reply:
I compiled and built the newest FreePastry.jar locally and then proceeded to upload it manually to archiva, as it does not happen automatically if i'm not mistaken. Was our bad that we still did not create relase 3.0.0 on git. And we chose 3.0.0 because of the use of Java 14, we thought that this might be seen as a "breaking change".